src/Network/Bolt/RoutingTable.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Laudis Neo4j package.
+ *
+ * (c) Laudis technologies <http://laudis.tech>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Laudis\Neo4j\Network\Bolt;
+
+use Bolt\Bolt;
+use Ds\Map;
+use Laudis\Neo4j\Formatter\BoltCypherFormatter;
+
+class RoutingTable
+{
+    private $roles = [
+        'leader' => 'WRITE',
+        'follower' => 'READ',
+        'router' => 'ROUTE',
+    ];
+
+    private Map $routingTable;
+    private ?int $ttl = null;
+    private Bolt $bolt;
+
+    public function __construct(Bolt $bolt, BoltCypherFormatter $formatter)
+    {
+        $this->bolt = $bolt;
+        $this->formatter = $formatter;
+        $this->loadRoutingTable();
```

Comment:
This needs to be removed. We can be lazy about loading the routing table and might be accounting for the race condition